app/helpers/captcha_helper.rb

```diff
@@ -0,0 +1,7 @@
+module CaptchaHelper
+  def captcha_tags
+    tag.div(class: "h-captcha", data: { sitekey: HCaptcha.site_key }) do
+      tag.script(nil, async: true, defer: "defer", src: "#{HCaptcha.endpoint}/1/api.js")
+    end
+  end
+end
```

Comment:
Could you change this to be a ViewComponent (called `CaptchaTags`) please?

It's basically the same code but in a class that can be tested, rather than just a global method.